ecs-cli/modules/cli/local/up_app.go

```diff
@@ -159,14 +187,29 @@ func decryptSecrets(containerSecrets []*secrets.ContainerSecret) (envVars map[st
 	return
 }
 
-// upComposeFile starts the containers in the Compose config with the environment variables defined in envVars.
-func upComposeFile(config *composeV3.Config, envVars map[string]string) {
+// upCompose starts the containers in the Compose files with the environment variables defined in envVars.
+func upCompose(envVars map[string]string, basePath string, overridePaths []string) {
+	// Gather environment variables
 	var envs []string
 	for env, val := range envVars {
 		envs = append(envs, fmt.Sprintf("%s=%s", env, val))
 	}
+	// Need to add $PATH because of --build, see https://stackoverflow.com/a/55371721/1201381
+	envs = append(envs, fmt.Sprintf("PATH=%s", os.Getenv("PATH")))
+
+	// Gather command arguments
+	var b strings.Builder
+	b.WriteString(filepath.Base(basePath))
+	args := []string{"-f", basePath}
+	for _, p := range overridePaths {
+		b.WriteString(fmt.Sprintf(", %s", filepath.Base(p)))
+		args = append(args, "-f", p)
+	}
+	args = append(args, "up", "--build", "-d")
 
-	cmd := exec.Command("docker-compose", "-f", config.Filename, "up", "-d")
+	// Run the command with the environment variables and arguments
+	logrus.Infof("Using %s files to start containers", b.String())
```

Comment:
Shall we also inform users which override files are actually used to start containers?

Reply:
We do that, here is an example for this log statement:
`Using docker-compose.ecs-local.yml, docker-compose.ecs-local.override.yml, app.override.yml files to start containers`